src/rdkafka_msgset_reader.c

```diff
@@ -403,6 +411,7 @@ rd_kafka_msgset_reader_decompress (rd_kafka_msgset_reader_t *msetr,
                                             rkbufz,
                                             msetr->msetr_rktp,
                                             msetr->msetr_tver,
+                                            NULL,
```

Comment:
Add a comment saying that there are no aborted transaction support for MsgVersion < 2